index.html

```diff
@@ -0,0 +1,534 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/normalizer.css">
+
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+  integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous"/>
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav class="width100 flex">
+      <div class="container-icons flex">
+        <button class="humburger" type="button" name="button"><i class="fas fa-bars"></i></button>
+        <i class="fas fa-search"></i>
+        <h3>Space and cosmos</h3>
```

Comment:
- [ ] Please consider using all letters capital for the "SPACE & COSMOS" link to have the same UI with the original page, also instead of "and" you can use HTML entities.

![image](https://user-images.githubusercontent.com/60184961/90851355-1884e080-e37d-11ea-8abd-0da15cdb5fee.png)